autoload/test/go/gotest.vim

```diff
@@ -12,7 +12,8 @@ function! test#go#gotest#build_position(type, position) abort
     if !empty(name) | let name = '-run '.shellescape(name, 1) | endif
     return [name]
   elseif a:type == 'file'
-    return []
+    let path = fnamemodify(a:position['file'], ':h.')
+    return path != '.' ? ['./' . path . '/...'] : []
```

Comment:
I don't need the dot in `fnamemodify` though I do need to check if the path is equal to "." which is the current directory. Have updated my PR accordingly.